app/models/ability.rb

```diff
@@ -179,7 +179,7 @@ def image_abilities
 
   def item_abilities
     if can_manage_items?
-      can [:index, :show, :create, :update, :messages, :move_stockit_item_set, :move_set_partial_qty,
+      can [:index, :show, :create, :update, :messages, :move_set_partial_qty,
```

Comment:
can we get rid of `move_set_partial_qty` as well?

Reply:
oh, forgot to remove it from the list. Good catch